### PR TITLE
Update template_weekly_scrum.md

### PR DESCRIPTION
**Pull request by ThijmenStaatsen**
# Pull Request Template

## Description
ik heb wat kleins veranderd in sprint doc

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

## Checklist:

Before submitting your PR, please review the following checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

## Additional Information

Any additional information, configuration, or data that might be necessary to reproduce the issue.

### CODE REVIEW

File: scrum/template_weekly_scrum.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 - [Insert Sprint Goal]
 
-### User Stories
+### Issues
```
Comment (Bobmarinus) [inline scrum/template_weekly_scrum.md:15]: ziet er goed uit broer neef

**Comment by Bobmarinus** [comment]
je bolle komkommer oma

File: scrum/template_weekly_scrum.md (hunk 1 above)
Comment (FabianDumitrascu) [reply scrum/template_weekly_scrum.md:15]: yes very nice

**Review by FabianDumitrascu — APPROVED** [review]
very good

**Pull request merged**